pages/osx/lp.md

```diff
@@ -0,0 +1,15 @@
+# lp
+
+> print files
+
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
```

Comment:
I'd suggest replacing

> (found in Printers & Scanners --> Options and Supplies)

with

> (use `lpstat -p | awk '{print $2}'` to list available printer names)

by the way, I don't think this command (lp) is osx-specific. I know that it works at least in Linux as well, so probably it should be moved to the "common" folder